<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>935</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="valider_gvi_PB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vérifier les informations</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="chantier_GB">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Chantier</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QGroupBox" name="moteur_GB">
         <property name="title">
          <string>Moteur</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="type_moteur_LBL">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Type moteur</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="sn_moteur_TOEDIT">
            <property name="maxLength">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="type_moteur_TOEDIT">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="sn_moteur_LBL">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SN moteur</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="5">
        <widget class="QGroupBox" name="operateurs_GB">
         <property name="title">
          <string>Opérateur(s)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="nom_operateurs_LBL">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nom du ou des opérateur(s)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nom_operateurs_TOEDIT"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="current_user_LBL">
            <property name="text">
             <string>Utilisateur Actuel :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="current_user_TOEDIT">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="n_lancement_TOEDIT">
         <property name="maxLength">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="client_TOEDIT">
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="n_lancement_LBL">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Numéro de lancement</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="date_LBL">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="date_TOEDIT">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CURRENT_ DATE</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="5">
        <widget class="QPushButton" name="valider_informations_chantier_PB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Prendre en compte les informations du chantier</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="client_LBL">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Client</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">QProgressBar
{
	text-align:center;
	border:2px solid white;
	border-radius:10px;
}
QProgressBar::Chunk
{
	background-color:rgb(0, 255, 0);
	border-radius:5px;
	margin:2px;
}
</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QScrollArea" name="liste_equipements_QSA">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1252</width>
         <height>493</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCharger"/>
    <addaction name="separator"/>
    <addaction name="actionImprimer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos_de_GVInspector"/>
    <addaction name="separator"/>
    <addaction name="actionA_propos_de_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionOuvrir_l_aide_de_GVIsor"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAutoriser_les_modifications"/>
    <addaction name="actionBloquer_les_modifications"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger une GVI...</string>
   </property>
  </action>
  <action name="actionA_propos_de_GVInspector">
   <property name="text">
    <string>A propos de General Visual Inspector...</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt">
   <property name="text">
    <string>A propos de Qt...</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="text">
    <string>Imprimer des documents...</string>
   </property>
  </action>
  <action name="actionOuvrir_l_aide_de_GVIsor">
   <property name="text">
    <string>Ouvrir l'aide de GVIsor...</string>
   </property>
  </action>
  <action name="actionAutoriser_les_modifications">
   <property name="text">
    <string>Autoriser les modifications...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionBloquer_les_modifications">
   <property name="text">
    <string>Bloquer les modifications...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>n_lancement_TOEDIT</tabstop>
  <tabstop>sn_moteur_TOEDIT</tabstop>
  <tabstop>client_TOEDIT</tabstop>
  <tabstop>nom_operateurs_TOEDIT</tabstop>
  <tabstop>valider_informations_chantier_PB</tabstop>
  <tabstop>valider_gvi_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
